pkg/tcpip/stack/ndp.go

```diff
@@ -251,6 +255,18 @@ type NDPDispatcher interface {
 	OnDHCPv6Configuration(tcpip.NICID, DHCPv6ConfigurationFromNDPRA)
 }
 
+// NDPRouterConfiguration is the NDP configuration for Router Advertisements
+type NDPRouterConfiguration struct {
+	RouterLifetime                     time.Duration
```

Comment:
Lets represent the lifetime fields in units of seconds so we can use the value directly in the Router Advertisement without worrying about what to do with values like 2.5s which are not representable in the RA.

Here and below w/ `PreferredLifetime`, `ValidLifetime`,